jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -169,6 +169,8 @@ public final void handleTextMessage(WebSocketSession session, TextMessage textMe
             if (event.getType() != null) {
                 if (event.getType().equalsIgnoreCase(EventType.IM_OPEN.name())) {
                     slackService.addDmChannel(event.getChannelId());
+                } else if (event.getType().equalsIgnoreCase(EventType.IM_CREATED.name())) {
+                    slackService.addDmChannel(event.getChannel().getId());
```

Comment:
I think we should use `event.getChannelId()` instead of `event.getChannel.getId()` as we may not get channel object in all events from slack.